2 - RTB/Documentazione interna/Verbali/24-02-11/24-02-11.typ

```diff
@@ -0,0 +1,106 @@
+- Revisione ultimi capitoli rimasti.
+
+
+=== Revisione #adr
+
+Il gruppo si è impegnato a risolvere tutte le criticità riscontrate dal #cardin durante la prima parte di revisione RTB, precisamente la valutazione è stata:
+- Il glossario va riferito anche con il numero di versione del documento;
+- UC2: quali sono le informazioni che possono essere modificate?
+- UC3.1: quali sono i dati necessari alla creazione dello scaffale?
+- UC3.2: cosa può essere modificato di uno scaffale?
+- UC4.1: quali sono i dati necessari alla creazione di un bin?
+- UC4.2: cosa può essere modificato di un bin?
+- UC6: il database è interno all'applicazione o è possibile collegarsi a database esterni? Se siì, quali? Se invece è interno, allora il database non può essere un attore secondario (in ogni caso, la primitiva grafica scelta non esiste in UML 2);
+- UC6.1 è precondizione di UC6, non sua inclusione;
+- UC6.1 cosa si intende con "configurazione"? Quali informazioni?
+- UC7: cosa sono le API? Se sono API interne, allora non possono essere attore secondario. Inoltre, perché la funzionalità è una "richiesta"?
+- UC8 e seguenti: modificare "interrogazione" con "visualizzazione". Quali informazioni sono visualizzate?
+- UC11 va analizzato più in dettaglio. I requisiti di qualità mancano della creazione dei manuali richiesti;
+- VM1 e VM2 sono funzionali;
+- VM3 non è chiaro e quindi non verificabile;
+- Non è presente il tracciamento casi d'uso / requisiti.
+
+Il documento abbozza una struttura corretta, ma il livello di analisi raggiunto è spesso poco dettagliato e lacunoso. Inoltre, sono presenti alcuni errori di modellazione.\
+Tutti queste criticità sono state corrette durante lo sprint appena concluso e il documento è ora pronto per la revisione prima del PB.
+
+=== Revisione #pdp e #pdq
+
+Il lavoro sul #pdp segue dal big bang attuato durante gli sprint precedenti e viene concluso. Viene inoltre redatto:
+- preventivo sprint 12;
+- preventivo sprint 13;
+- consuntivo sprint 13;
+- preventivo sprint 14;
+segue un generale refactoring del documento.
+
+Il #pdq invece è stato esteso e aggiornato con le metriche del periodo e la loro analisi. Vengono inoltre inseriti i grafici generati con Grafana.
+
+=== Candidatura seconda parte RTB
+
+Il gruppo ha concordato l'invio della mail per sostenere la seconda parte della revisione esterna RTB con il #vardanega per la data 14/02/2024.
+
+== Analisi di retrospettiva
+
+=== Keep doing
+
+A contribuire al successo dello sprint 14 troviamo un efficace pianificazione delle task e il rispetto della modalità di lavoro asincrona.
+
+=== Improvements  
+
+Tra le criticità evidenziate troviamo:
++ un basso utilizzo di Miro per i meeting di retrospettiva;
++ ritardi nelle correzioni delle pr;
++ struttura dei verbali non sempre conforme.
+
+=== Soluzioni adottate
+Le soluzioni concordate dal gruppo in relazione alle cricità trovate durante la retrospettiva sono:
++ maggiore partecipazione su Miro:\
```

Comment:
Capisco l'intento di creare una corrispondenza tra improvments e soluzioni, ma anche qui si va contro alla struttura dei precedenti verbali. E' ragionevole come cosa, anche se non normata e non presente in nessun verbale precedente.